pubspec.yaml

```diff
@@ -44,11 +44,10 @@ dependencies:
   dio_smart_retry: ^5.0.0
   dio: ^5.0.0
   dynamic_color: ^1.4.0
-  extended_image: ^8.0.1
   file_picker: ^5.2.3
   flutter_cache_manager: ^3.0.0-nullsafety.1
   flutter_displaymode: ^0.6.0
-  flutter_downloader: ^1.9.0
+  flutter_downloader: ^1.10.4
```

Comment:
Bumped this requirement because app build was getting crazy. Perhaps it's not required.

But I got an error saying that Enumerator was somehow invalid.